src/extension/sldService/src/main/java/org/geoserver/sldservice/utils/classifier/RulesBuilder.java

```diff
@@ -86,7 +92,11 @@ public void setIncludeStrokeForPoints(boolean includeStrokeForPoints) {
             String functionName) {
         try {
             final Function classify =
-                    FF.function(functionName, FF.property(property), FF.literal(classNumber));
+                    FF.function(
```

Comment:
Problematic with that function having two optional values at the end (see the GeoTools review)

Reply:
fixed